mods/cssb/abilities.js

```diff
@@ -184,4 +184,31 @@ exports.BattleAbilities = {
 		},
 		desc: "If the user is statused, their SpA, SpD, and Spe is multiplied by 1.5x.",
 	},
+	// Redroller
+	"graveyard": {
+		id: "graveyard",
+		name: "Graveyard",
+		onStart: function (pokemon, target, effect) {
+			this.addPseudoWeather('graveyard', pokemon);
+			this.useMove('willowisp', pokemon);
+			this.boost({def: 2, spd: 2});
+		},
+		effect: {
+			duration: 0,
+			onStart: function (target, source) {
+				this.add('-fieldstart', 'move: Trick Room', '[of] ' + source);
+				this.getStatCallback = function (stat, statName) {
+					// If stat is speed and does not overflow (Trick Room Glitch) return negative speed.
+					if (statName === 'spe' && stat <= 1809) return -stat;
+					return stat;
+				};
+			},
+			onResidualOrder: 23,
+			onEnd: function () {
+				this.add('-fieldend', 'move: Trick Room');
+				this.getStatCallback = null;
+			},
+		},
+		desc: "On switch-in, user activates permanent trick room, uses will-o-wisp, gains 2x def and spd.",
```

Comment:
This is very nit-picky, but can you please upper case properly and completely spell Defense and Special Defense (and make a shortDesc for abbreviations (assuming we want to be like main?))

Reply:
@DeathlyPlays it doesnt fit if they are spelled out

Reply:
and !dt ability name ONLY shows regular desc

Reply:
Actually it shows shortdesc if there is one. But desc if no shortdesc